default/scripting/species/common/general.macros

```diff
@@ -23,6 +23,7 @@ STANDARD_METER_GROWTH
             activation = And [
                 Planet
                 TargetPopulation high = 0
+                Population high = 0.1
             ]
             effects = [
                 SetResearch value = 0
```

Comment:
I just remembered about #2184.  If you have this also zero out the target meters here, then I believe that #2184 would also be resolved (I am not positive that there are no other sources of such zeroing, but this is the only one I recall).